src/engine/Anchor.ts

```diff
@@ -0,0 +1,58 @@
+import { Vector, vec } from "./excalibur";
+
+export class Anchor {
+  /**
+   * get TopLeft Anchor Vector
+   */
+  public static get TopLeft(): Vector {
```

Comment:
The Vector class already has a set of these static functions, so why should these be in a new class? https://github.com/excaliburjs/Excalibur/blob/main/src/engine/Math/vector.ts#L16

Vector doesn't contain getters for diagonal vectors yet, so I'd love to see yours added to it.